blocks/api/raw-handling/utils.js

```diff
@@ -63,7 +68,7 @@ const whitelist = {
 };
 
 export function isWhitelisted( element ) {
-	return !! whitelist[ element.nodeName.toLowerCase() ];
+	return !! whitelist.hasOwnProperty( element.nodeName.toLowerCase() );
```

Comment:
The type coercion is unnecessary (`hasOwnProperty` will certainly return a boolean).